### PR TITLE
Gursimar Saini

### PR DESCRIPTION
**Pull request by proxyflux**
NOTE:
- Please fill out the template below 

## Did you read the instructions in the [README](https://github.com/the-ethan-hunt/first-timers-guide/blob/master/README.md)?

> Y

## Have you read the instructions in [CONTRIBUTING](https://github.com/the-ethan-hunt/first-timers-guide/blob/master/CONTRIBUTING.md)?

> Y

## Have you written this article by yourself? Have you taken inspiration from anywhere and then written it in your words?

> Used the features part from wikipedia

### CODE REVIEW

File: guides/databases/phpmyadmin.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+- Import data from CSV and SQL
+
+- Export data to various formats: CSV, SQL, XML, PDF (via the TCPDF library), ISO/IEC 26300 OpenDocument Text and Spreadsheet, Word, Excel, LaTeX and others
+
+- Administering multiple servers
+
+- Creating PDF graphics of the database layout
+
+- Creating complex queries using query-by-example (QBE)
```
Comment (the-ethan-hunt) [inline guides/databases/phpmyadmin.md]: What does QBE mean? Explain it.

File: guides/databases/phpmyadmin.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+---
+
+>**PHPMyAdmin** is a free software tool written in PHP, intended to handle  the administration of MySQL over the Web. phpMyAdmin supports a wide range  of operations on MySQL and MariaDB. Frequently used operations (managing  databases, tables, columns, relations, indexes, users, permissions, etc) can  be performed via the user interface, while you still have the ability to  directly execute any SQL statement.
+
+### Features provided by the program include:
+
+
+- Web interface
+
+- MySQL and MariaDB database management
```
Comment (the-ethan-hunt) [inline guides/databases/phpmyadmin.md]: How can I benefit from Maria DB database management? Explain whatever you state in a succinct way. 😄

**Review by dhruvapte26 — CHANGES_REQUESTED** [review]

File: guides/databases/phpmyadmin.md (hunk 1 above)
Comment (GursimarSaini) [reply guides/databases/phpmyadmin.md]: It's a query language used in relational databases that help users to perform queries by providing a simple UI. It is just an abstraction between the user and the real query that the database system will receive and will transform into a DML form such as SQL.

File: guides/databases/phpmyadmin.md (hunk 2 above)
Comment (GursimarSaini) [reply guides/databases/phpmyadmin.md]: Mostly we can say it's just the same as MySQL, but it's kind of a forked version of MySQL with advanced features like: --it's better at handling both large and small scale data better than former. 
-MariaDB has had a significant boost in terms of security features like internal security and password check, PAM and LDAP authentication, Kerberos, user roles, and strong encryption over tablespaces, tables and logs.
-MySQL has no support for extensibility, but MariaDB is built to be extensible at every layer—client, cluster, kernel and storage.
And a lot others, https://www.google.com/amp/s/opensourceforu.com/2018/04/why-mariadb-scores-over-mysql/amp/

**Pull request closed without merging**